webapp/src/selectors.js

```diff
@@ -52,11 +52,11 @@ function mapPrsToDetails(prs, details) {
 export const getSidebarData = createSelector(
     getPluginState,
     (pluginState) => {
-        const {username, sidebarContent, reviewDetails, yourPrDetails, organization, rhsState} = pluginState;
+        const {username, sidebarContent, reviewsDetails, yourPrsDetails, organization, rhsState} = pluginState;
```

Comment:
What's the reason for these changes?

Reply:
Previously it was wrong now i have corrected it.

Reply:
I think the previous names were better.